principles.md

```diff
@@ -0,0 +1,25 @@
+## Product and Engineering Principles
+
+**Simpler always wins in the end**
+
+We choose simpler designs whenever possible. Having less configuration knobs
+is better for security. Complex products are easier to hack and harder
+to use. Security feature is only good, when it's easy to use.
+
+That's why we stick to [client certificates](https://goteleport.com/teleport/how-it-works/certificate-based-authentication-ssh-kubernetes/)
+and as the only way to authenticate and [reduce the number of ports and protocols](https://github.com/gravitational/teleport/issues/5777).
+
+**Security through transparency**
+
+The only way to build secure systems is through peer review and transparency.
```

Comment:
```suggestion
The most reliable way to build secure systems is through peer review and transparency.
```

Reply:
I'd argue that's the only way to build a secure system, not just the most reliable :)

Reply:
I was thinking technically you can build trivial systems that are secure, but that's a rather pedantic point, plus any example I try and come up with stands on the shoulders of a non-trivial system built on peer review.